src/main/java/de/tum/in/www1/artemis/service/ExerciseImportService.java

```diff
@@ -62,6 +62,7 @@ protected void copyExerciseBasis(final Exercise newExercise, final Exercise impo
         newExercise.setDifficulty(importedExercise.getDifficulty());
         newExercise.setGradingInstructions(importedExercise.getGradingInstructions());
         newExercise.setGradingCriteria(importedExercise.copyGradingCriteria(gradingInstructionCopyTracker));
+        newExercise.setCompetencies(importedExercise.getCompetencies());
```

Comment:
What is supposed to happen when I import an exercise from another course? e.g. I imported an exercise from a different course that had two linked competencies. When viewing the exercise in my course, the linked competencies were still there, even though they are inaccessible in my course.

Then, when I tried to delete the other course, I got an internal server error because the competencies are still referenced in the other course:

    "detail": "could not execute batch [Batch entry 0 delete from competency where id=('97'::int8) was aborted: ERROR: update or delete on table \"competency\" violates foreign key constraint \"FKsqsxg20yruu16ca9x383wnhci\" on table \"competency_exercise\"\n  Detail: Key (id)=(97) is still referenced from table \"competency_exercise\".  Call getNextException to see other errors in the batch.] [delete from competency where id=?]; SQL [delete from competency where id=?]; constraint [FKsqsxg20yruu16ca9x383wnhci]",
    "path": "/api/admin/courses/74",

Reply:
They should not be imported. Fixed!